buildlib/azure-pipelines-pr.yml

```diff
@@ -288,17 +288,25 @@ stages:
               downloadPath: $(System.DefaultWorkingDirectory)
           - bash: chmod u+rwx $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId) -R
           - bash: |
-              set -x
-              ./bin/ucx_info -e -u t 2>&1 | tee info.txt
-              grep -i error info.txt
-              retVal=$?
-              if [ $retVal -eq 0 ]; then
-                  exit 1;
-              fi
-              exit 0;
+              set -xEe
+              check_ucx_info() {
+                 ./bin/ucx_info -e -u t $@ |& tee ucx_info.txt
+                 ## the word "lane" should be found
+                 egrep 'lane' ucx_info.txt
+                 ## no failures
+                 ! egrep -i 'error|fail|warn' ucx_info.txt
```

Comment:
maybe better check for `fatal` (assert also goes here) instead of `fail`?

Reply:
added assert and fatal